sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/sonar/LangAttributeCheck.java

```diff
@@ -39,15 +39,31 @@ private static boolean isHtmlTag(TagNode node) {
   private static boolean hasLangAttribute(TagNode node) {
     return node.hasProperty("lang")
       || node.hasProperty("xml:lang")
-      || hasLangAttributeFromWordPress(node);
+      || hasWordPressLangAttribute(node)
+      || hasThymeleafLangAttribute(node);
   }
 
   /**
    * Using WordPress, HTML attributes can be set using the php function `language_attributes`
    */
-  private static boolean hasLangAttributeFromWordPress(TagNode node) {
```

Comment:
Renaming to match function name from [SONARHTML-140](https://sonarsource.atlassian.net/browse/SONARHTML-140)